Create code_of_conduct.md

rzr opened the pull request:
Origin: https://github.com/SiliconLabsSoftware/agreements-and-guidelines/commit/ca3aba97c9b45a7d00547cef9232ba92301e5330.patch

silabs-akosj commented:
@rzr 
Instead of duplicating the content, please use the following wording and link to point to the code of conduct file in the agreements and guidelines repo:

# Silicon Labs Community Terms of Use

Please read the full Terms of Use and Code of Conduct in the [agreements-and-guidelines](https://github.com/SiliconLabsSoftware/agreements-and-guidelines) repository: [Code of Conduct](https://github.com/SiliconLabsSoftware/agreements-and-guidelines/blob/main/code_of_conduct.md)

silabs-akosj reviewed (changes requested):
Please use this template instead:

# Silicon Labs Community Terms of Use

Please read the full Terms of Use and Code of Conduct in the [agreements-and-guidelines](https://github.com/SiliconLabsSoftware/agreements-and-guidelines) repository: [Code of Conduct](https://github.com/SiliconLabsSoftware/agreements-and-guidelines/blob/main/code_of_conduct.md)

rzr commented:
> Please use this template instead:
> # Silicon Labs Community Terms of Use
> 
> Please read the full Terms of Use and Code of Conduct in the [agreements-and-guidelines](https://github.com/SiliconLabsSoftware/agreements-and-guidelines) repository: [Code of Conduct](https://github.com/SiliconLabsSoftware/agreements-and-guidelines/blob/main/code_of_conduct.md)

This mean it could change over time ? , should I pin the file revision of use main branch ?

silabs-akosj commented:
> This mean it could change over time ? , should I pin the file revision of use main branch ?
Yes, the idea is that Silabs will maintain one source of truth document in a central repository and with this link we would like to point to the latest approved version on the main branch.

rzr commented:
> > This mean it could change over time ? , should I pin the file revision of use main branch ?
> > Yes, the idea is that Silabs will maintain one source of truth document in a central repository and with this link we would like to point to the latest approved version on the main branch.

IMHO it is not a good practice , a contract that change over time, is just like signing a blank page.
but no problem I can align to the policy in place.

rzr committed:
fix(community): Add CoC from vendor

I note that those links are floating.

Origin: https://github.com/SiliconLabsSoftware/z-wave-nvm-migration-tool/pull/39
Signed-off-by: Philippe Coval <philippe.coval@silabs.com>
